hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -1471,7 +1471,7 @@
         <xs:complexType>
             <xs:complexContent>
                 <xs:extension base="hazelcast-bean">
-                    <xs:all>
+                    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
good one. I copied it from [here](https://github.com/jerrinot/hazelcast/blob/3bf3ffbf6661f8ca4d7bc079d95ac14091299df0/hazelcast-client/src/main/resources/hazelcast-client-config-3.9.xsd#L26-L26) Perhaps one can have a very basic configuration where everything is on default.